### PR TITLE
Add Reolink sensor platform

### PR DESCRIPTION
**Pull request by starkillerOG**
## Proposed change

Add Reolink sensor platform, see https://github.com/home-assistant/core/pull/96323

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/96323
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/reolink.markdown [hunk 1]
```diff
@@ -54,9 +54,16 @@ Depending on the supported features of the camera, binary sensors are added for:
...
+These sensors receive events using 3 methods in order: ONVIF push, ONVIF long polling or fast polling (every 5 seconds).
+The latency for receiving the events is the best for ONVIF push and the worst for fast polling, the fastest available method that is detected to work will be used, and slower methods will not be used. The `Event Connection` sensor shows the current method being used.
+As a extra redundancy these sensors are polled every 60 seconds together with the update of all other entities.
 Not all camera models generate ONVIF push events for all event types, some binary sensors might, therefore, only be polled.
 For list of Reolink products that support ONVIF see the [Reolink Support Site](https://support.reolink.com/hc/en-us/articles/900000617826).
+If the `Event Connection` sensor does not show `ONVIF push` and you wish to reduce the latency, see the troubleshooting section.
+
+## General notes for all entities
+
+Entities listed below that have a * behind their name are disabled by default. To enable them go to settings->devices->reolink->device->+x enties not shown, click the entity you want to enable, settings (gear) -> enable.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
```
I believe this section isn't required in this PR as it is added in #28170

File: source/_integrations/reolink.markdown [hunk 2]
```diff
@@ -155,6 +162,13 @@ When the floodlight entity is ON always ON, when OFF controlled based on the int
 
 When IR light entity is OFF always OFF, when ON IR LEDs will be on when the camera is in night vision mode, see the "Day night mode" select entity.
 
+## Sensor entities
+
+Depending on the supported features of the camera, sensor entities are added for:
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
Depending on the supported features of the camera, the following sensor entities are added:
```

File: source/_integrations/reolink.markdown [hunk 3]
```diff
@@ -54,9 +54,16 @@ Depending on the supported features of the camera, binary sensors are added for:
 - AI pet detection
 - AI face detection
 
-These sensors are polled every 60 seconds and receive ONVIF push events for immediate updates.
+These sensors receive events using 3 methods in order: ONVIF push, ONVIF long polling or fast polling (every 5 seconds).
+The latency for receiving the events is the best for ONVIF push and the worst for fast polling, the fastest available method that is detected to work will be used, and slower methods will not be used. The `Event Connection` sensor shows the current method being used.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
The latency for receiving the events is the best for ONVIF push and the worst for fast polling. The fastest available method that is detected to work will be used. Slower methods will not be used. The `Event Connection` sensor shows the current method being used.
```

File: source/_integrations/reolink.markdown [hunk 4]
```diff
@@ -54,9 +54,16 @@ Depending on the supported features of the camera, binary sensors are added for:
...
+As a extra redundancy these sensors are polled every 60 seconds together with the update of all other entities.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
For redundancy, these sensors are polled every 60 seconds together with the update of all other entities.
```

File: source/_integrations/reolink.markdown [hunk 5]
```diff
@@ -54,9 +54,16 @@ Depending on the supported features of the camera, binary sensors are added for:
...
 Not all camera models generate ONVIF push events for all event types, some binary sensors might, therefore, only be polled.
 For list of Reolink products that support ONVIF see the [Reolink Support Site](https://support.reolink.com/hc/en-us/articles/900000617826).
+If the `Event Connection` sensor does not show `ONVIF push` and you wish to reduce the latency, see the troubleshooting section.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
If the `Event Connection` sensor does not show `ONVIF push` and you wish to reduce the latency, refer to the [troubleshooting](#troubleshooting) section.
```
Is there a section on latency in the troubleshooting section? To me, it reads more like general network troubleshooting. Would it make sense to maybe explicitly mention the effect on latency in the troubleshooting step itself?

**Review by c0ffeeca7 — DISMISSED** [review]

File: source/_integrations/reolink.markdown (hunk 3 above)
Comment (frenck) [reply source/_integrations/reolink.markdown]: As per the comment on the parent PR, this Sensor needs to be removed/reverted.

File: source/_integrations/reolink.markdown (hunk 1 above)
Comment (starkillerOG) [reply source/_integrations/reolink.markdown]: Correct, thanks

File: source/_integrations/reolink.markdown (hunk 3 above)
Comment (starkillerOG) [reply source/_integrations/reolink.markdown]: I have removed it.

File: source/_integrations/reolink.markdown (hunk 5 above)
Comment (starkillerOG) [reply source/_integrations/reolink.markdown]: I have removed this sentence now that the Event Connection sensor has been removed.
Will add something about latency in a seperate docs PR.

Comment (starkillerOG) [reply source/_integrations/reolink.markdown]: I added new documentation about this here: https://github.com/home-assistant/home-assistant.io/pull/28288

**Review by frenck — APPROVED** [review]
Thanks, @starkillerOG 👍 

../Frenck

**Pull request merged**